test/integration/thruster.cc

```diff
@@ -331,13 +331,19 @@ void ThrusterTest::TestWorld(const std::string &_world,
     EXPECT_NEAR(0.0, angVel.Y(), _baseTol);
     EXPECT_NEAR(0.0, angVel.Z(), _baseTol);
   }
-
-  modelPoses.clear();
-  propellerAngVels.clear();
-  propellerLinVels.clear();
+  if(!modelPoses.empty())
```

Comment:
This can all be simplified by `ASSERT_FALSE(modelPoses.empty())` as @mjcarroll suggested. No need to throw an exception as that will be handled by `ASSERT_FALSE`.